perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -404,6 +429,16 @@ def _ProcessMultiStreamResults(raw_result, operation, sizes,
         'Multi-stream ' + operation + ' ' + name,
         value.magnitude, str(value.units), metadata=distribution_metadata))
 
+  # QPS metrics
+  results.append(sample.Sample(
+      'Multi-stream ' + operation + ' QPS',
```

Comment:
Do you want to append a 'any streams active' here?  Or are you trying to keep it backward compatible?